tensor_api.ipynb

```diff
@@ -0,0 +1,563 @@
+{
```

Comment:
<p>Write more description about slicing/indexing. </p><p>Most importantly, mention that they produce a "view" of the tensor. Demonstrate that if <code>b = a[1:]</code>  and if <code>b</code> is modified, <code>a</code> will be modified as well and vise-versa.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/intel-isl/Open3D/pull/2067/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='22'/>